src/gt4py/next/common.py

```diff
@@ -301,15 +309,14 @@ def is_int_index(p: Any) -> TypeGuard[IntIndex]:
 
 def is_named_range(v: AnyIndexSpec) -> TypeGuard[NamedRange]:
```

Comment:
in the end this function can be removed and replaced by just an isinstance check